Vagrantfile

```diff
@@ -29,6 +29,9 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.provision :shell, :inline => <<-EOS
     # To prevent "dpkg-preconfigure: unable to re-open stdin: No such file or directory" warnings
     export DEBIAN_FRONTEND=noninteractive
+    # Make sure git submodules are checked out!
+    apt-get -qq install -y git >/dev/null
+    git submodule --quiet update --init --recursive
```

Comment:
Alas, this still doesn't work because it needs to be run inside the `fixmystreet` directory. But even then... checking out the submodule from within the vagrant guest ends up breaking the git repo on the host because `git-submodule` uses absolute paths within its config files.

So I figured there must be a way to do the submodule check out within the vagrant box and then update the config paths to refer to absolute paths on the host. Turns out you can pass arguments to the provisioning script, so [here's a Vagrantfile](https://gist.github.com/davea/95954eb34c531734eeca) with an updated provisioning script that receives the absolute path from the host and rejigs the git config files to un-break the repo on the host. Should do the trick, both for repos that have been cloned recursively and those that haven't.